cost:
  kj: 0.01
  kt: 1.0
  kd: 0.5
  kd_v: 0.5
  klat: 1.0
  klon: 2.0

path:
  max_speed: 13.89
  max_accel: 2.0
  max_curvature: 1.0
  max_road_width: 10.0
  d_road_w: 1.0
  dt: 0.2
  maxt: 5.0
  mint: 2.0
  target_speed: 8.33
  d_t_s: 1.389
  n_s_sample: 1.5
  robot_radius: 2.0
  min_lat_vel: -1.0 
  max_lat_vel: 1.0
  d_d_ns: 0.5
  max_shift_d: 3

waypoints:
  # W_X: [38, 38, 38, 38, 38, 38, 38, 38]
  # W_Y: [-57, -45, -32.0, -18.5, -12.0, 0.0, 12, 35]
  W_X_0: [88.7100448608, 79, 68, 59, 56, 47]
  W_Y_0: [-237.4, -237.4, -237.4, -237.4, -237.4, -237.4]
  initial_orien_0: [4.29748423362e-05, -2.98275509289e-05, 0.999999998626, 3.46339124346e-06]
  # position: 
  #     x: 232.390625,232.430603027,232.404830933,232.357696533,232.302032471, 232.066543579
  #     y: 54.8937492371,41.1463088989,8.6428976059,-5.94234800339,-24.8151035309, -63.8651046753
  #     z: 0.00589141855016
  #   orientation: 
  #     x: -0.000151316969895
  #     y: -9.23353324922e-05
  #     z: -0.705579505329
  #     w: 0.70863074322
  #not working yet
  W_X_1: [ 232.4, 232.4, 232.4, 232.4, 232.4, 232.4]
  W_Y_1: [ -54.9, -41.1, -8.6, 5.9, 24.8, 63.9]
  initial_orien_1: [-0.000151316969895, -9.23353324922e-05, -0.705579505329, 0.70863074322]

  #working z=4
  W_X_2: [129.2, 125.4, 117.2, 110.6, 100.1, 89.1]
  W_Y_2: [-187.5, -187.5, -187.7, -187.8, -187.9, -188.1]
  initial_orien_2: [0.00209806614468, 9.41751505822e-05, -0.999960967881, 0.00858207219225] 

  #working z=4
  W_X_3: [-7.0, -6.8, -6.6, -4.6, -2.8, -0.8]
  W_Y_3: [-149.9, -131.9, -118.9, -115.7, -113.4, -111.0]
  initial_orien_3: [0.000100201765332, -0.000101843882083, 0.702664166886, 0.711521642792]

  #working z=4
  W_X_4: [169.4, 177.2, 181.6, 185.1, 186.5, 190.8]
  W_Y_4: [-108.4, -108.5, -108.4, -108.8, -109.2, -111.5]
  initial_orien_4: [-5.69329386381e-07, -0.000144361381856, -0.00484370353118, 0.999988258779]

  #working z=4
  W_X_5: [114.5, 119.0, 125.9, 130.6, 136.4, 135.7]
  W_Y_5: [-190.031, -190.4, -191.1, -193.4, -202.5, -206.7]
  initial_orien_5: [-0.00694699000478, -0.00266947421288, -0.0288480176913, 0.999556104034]

  #worling z=4
  W_X_6: [120.6, 141.5, 143.4, 145.6, 150.9, 164.8]
  W_Y_6: [-187.4, -187.7, -188.3, -188.9, -188.9, -188.9]
  initial_orien_6: [-4.51591738068e-07, 0.000110029252463, 0.00461763856976, 0.999989332597]

  #worling z=4
  W_X_7: [158.8, 161.2, 162.7, 167.2, 167.5, 169.8]
  W_Y_7: [-191.1, -190.8, -190.3, -189.2, -189.0, -188.6]
  initial_orien_7: [3.89114099321e-06, 0.000110199925246, -0.0347880277099, 0.999394707295]

  #working z=4
  W_X_8: [179.1, 189.3, 189.4, 189.7, 189.4, 189.1]
  W_Y_8: [-192.3, -195, -206.2, -221.2, -232.2, -232.2]
  initial_orien_8: [0.000102712323876, -0.00362497202428, 0.0599469849639, 0.998194974953]

  #working z=2
  W_X_9: [193.1, 193.1, 193.1, 193.1, 193.1, 193.1]
  W_Y_9: [-127.8, -139.7, -144.7, -145.7, -152.3, -160.0]
  initial_orien_9: [-8.89091465727e-05, -9.4793141613e-05, -0.698458344212, 0.71565070007]